pages/dkp/kaptain/2.0.0/install/prerequisites/index.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: layout.pug
+navigationTitle: Prerequisites
+title: Prerequisites for installing Kaptain
+menuWeight: 2
+excerpt: Ensure these prerequisites are met before installing Kaptain
+beta: false
+enterprise: false
+---
+
+- [`kubectl`][kubectl] on your installation machine 
+
+- A DKP cluster, either one of the following, depending on your license 
+
+  - An **Essential cluster** for customers with an [Essential license](../../../../kommander/2.2/licensing/essential/)
+  - A **Managed** or **Attached** cluster for customers with an [Enterprise licence](../../../../kommander/2.2/licensing/enterprise/), in which case, said cluster needs to be the only one in its workspace
```

Comment:
I think we use U.S. spelling for 'license', not British, right?   (We did in the previous line. 😄 )

```suggestion
  - A **Managed** or **Attached** cluster for customers with an [Enterprise license](../../../../kommander/2.2/licensing/enterprise/), in which case, said cluster needs to be the only one in its workspace
```

Reply:
Definitely yes 🙈